src/pages/Pages.tsx

```diff
@@ -13,19 +15,25 @@ const Supplement = lazy(async () => await import("./supplement/supplement"));
 
 function Pages() {
   return (
-    <Suspense fallback={<div>Loading...</div>}>
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/account" element={<Account />} />
-        <Route path="/cart" element={<Cart />} />
-        <Route path="/checkout" element={<Checkout />} />
-        <Route path="/contact" element={<Contact />} />
-        <Route path="/login" element={<Login />} />
-        <Route path="/quiz" element={<Quiz />} />
-        <Route path="/shop" element={<Shop />} />
-        <Route path="/supplement" element={<Supplement />} />
-      </Routes>
-    </Suspense>
+    <>
+      <Header />
+      <div className="container">
+        <Suspense fallback={null}>
```

Comment:
What does this fallback do if it is `null`? Just not render?

Reply:
It doesn't render anything